### PR TITLE
YD-455 Small code and copyright corrections

### PR DESCRIPTION
**Pull request by Bert-R**
* Code corrections were flagged by SonarQube
* Copyrights correct automatically through Eclipse plug-in

### CODE REVIEW

File: core/src/main/java/nu/yona/server/exceptions/EmailException.java [hunk 1]
```diff
@@ -1,9 +1,6 @@
 /*******************************************************************************
- * Copyright (c) 2016 Stichting Yona Foundation
```
Comment (aristotelos) [inline core/src/main/java/nu/yona/server/exceptions/EmailException.java]: There are some accidental format changes like this. Just to let you know.

File: core/src/main/java/nu/yona/server/goals/service/ActivityCategoryService.java [hunk 2]
```diff
@@ -225,9 +225,9 @@ private String getName(ActivityCategory entity)
 
 	/**
 	 * This method calls flush on the activity category repository. It shouldn't be necessary to do this, but the implicit flush
-	 * (done just before JPA sends a query to the database) causes a
```
Comment (aristotelos) [inline core/src/main/java/nu/yona/server/goals/service/ActivityCategoryService.java]: Formatting change here as well.

File: core/src/main/java/nu/yona/server/messaging/service/DisclosureResponseMessageDto.java [hunk 3]
```diff
@@ -51,8 +51,7 @@ public String getType()
 	@Override
 	public Set<String> getPossibleActions()
 	{
-		Set<String> possibleActions = super.getPossibleActions();
-		return possibleActions;
+		return super.getPossibleActions();
```
Comment (aristotelos) [inline core/src/main/java/nu/yona/server/messaging/service/DisclosureResponseMessageDto.java]: I think SonarQube will warn about this again :).

File: core/src/main/java/nu/yona/server/sms/SmsTemplate.java [hunk 4]
```diff
@@ -0,0 +1,21 @@
+package nu.yona.server.sms;
+
+public enum SmsTemplate
+{
+	ADD_USER_NUMBER_CONFIRMATION("add-user-number-confirmation"), CHANGED_USER_NUMBER_CONFIRMATION(
```
Comment (aristotelos) [inline core/src/main/java/nu/yona/server/sms/SmsTemplate.java]: This formatting is strange...

File: core/src/main/java/nu/yona/server/exceptions/EmailException.java (hunk 1 above)
Comment (Bert-R) [inline core/src/main/java/nu/yona/server/exceptions/EmailException.java]: These are reformatted when the file is updated. That's normal.

File: core/src/main/java/nu/yona/server/goals/service/ActivityCategoryService.java (hunk 2 above)
Comment (Bert-R) [inline core/src/main/java/nu/yona/server/goals/service/ActivityCategoryService.java]: This seems a slight difference in line length. That might be the cause of the accidental formatting difference. Please keep flagging them.

File: core/src/main/java/nu/yona/server/messaging/service/DisclosureResponseMessageDto.java (hunk 3 above)
Comment (Bert-R) [inline core/src/main/java/nu/yona/server/messaging/service/DisclosureResponseMessageDto.java]: Very true :)
I'll remove the method.

**Comment by Bert-R** [comment]
Removed the useless method.

**Pull request merged**